src/vs/platform/update/electron-main/updateService.win32.ts

```diff
@@ -99,90 +99,59 @@ export class Win32UpdateService extends AbstractUpdateService implements IRelaun
 		await super.initialize();
 	}
 
-	protected buildUpdateFeedUrl(quality: string): string | undefined {
-		let platform = `win32-${process.arch}`;
+	// --- START POSITRON ---
+	protected buildUpdateFeedUrl(channel: string): string | undefined {
+		const platform = `win/${process.arch === 'x64' ? 'x86_64' : 'arm64'}`;
+		const prefix = getUpdateType() === UpdateType.Setup ? 'system-' : 'user-';
+		const baseUrl = createUpdateURL(platform, channel, this.productService);
 
-		if (getUpdateType() === UpdateType.Archive) {
-			platform += '-archive';
-		} else if (this.productService.target === 'user') {
-			platform += '-user';
-		}
+		// TODO: properly determine if this is a user or system install
+		const url = `${baseUrl}/${prefix}releases.json`;
 
-		return createUpdateURL(platform, quality, this.productService);
+		return url;
 	}
 
-	protected doCheckForUpdates(context: any): void {
```

Comment:
It looks like this whole function from upstream got refactored out. Could you leave it in place so that it doesn't generate merge conflicts when it changes upstream? It's cool to mark it with `@ts-ignore` if it becomes unused.